webpack.config.js

```diff
@@ -1,42 +1,63 @@
 const path = require('path');
+const fs = require('fs');
 const NodePolyfillPlugin = require('node-polyfill-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
 
 const DIST = path.resolve(__dirname, 'public');
 
-module.exports = {
-  devtool: 'inline-source-map',
-  mode: 'development',
-  entry: './src/index.ts',
-  output: {
-    filename: 'bundle.js',
-    path: DIST,
-    publicPath: DIST,
-  },
-  devServer: {
-    contentBase: DIST,
-    port: 9011,
-    writeToDisk: true,
-  },
-  plugins: [
-    new NodePolyfillPlugin({
-      excludeAliases: ['stream', 'buffer'],
-    }),
-  ],
-  module: {
-    rules: [
-      {
-        test: /\.tsx?$/u,
-        use: 'ts-loader',
-        exclude: /node_modules/u,
-      },
+// eslint-disable-next-line node/no-sync
+const appDirectory = fs.realpathSync(process.cwd(), 'utf8');
+const resolveApp = (relativePath) => path.resolve(appDirectory, relativePath);
+// eslint-disable-next-line import/no-dynamic-require
+const prefix = require(resolveApp('package.json')).homepage;
```

Comment:
This seems unnecessary :thinking: `require` imports relative to the current module rather than the current directory. So the only effect here is to break the build unless you're in the root repo directory.

Unless I'm missing something, this should work more reliably:

```suggestion
// eslint-disable-next-line import/no-dynamic-require
const prefix = require('./package.json')).homepage;
```